ada_2_fhir-r4/zibs2024/ada_instance/ADA_GUIDELINES.md

```diff
@@ -0,0 +1,11 @@
+# ADA guidelines
+
+In this folder instances of the zib scenarios are added in ADA format, which are used to test the conversion from zibs to FHIR resources.
+
+## ADA environment
+All ADA instances are created via the ADA environment accessible at `https://decor.nictiz.nl/art-decor/ada-data/projects/zib2024-prerelease/views/zorginformatiebouwstenen_zib_2024_index.xhtml`. Make sure to add these while being logged in with the _zibs2024ada_ account. This ensures that instances that have been created in the context of this project can be easily distinguished from other instances, while also making it easy for anyone in the zibs2024 project to update existing instances (without copying existing ones made by other accounts).
```

Comment:
We might also consider to use the existing _zibsada_ account, which is currently used for the BgZ and BgZ-MSZ ADA instances. However, if we want to make sure only our team can edit the zibs2024 ADA instances, creating a dedicated new account is a better approach.

Reply:
Let's try our custom account for now.